### PR TITLE
WIP Update s390 docker image for jdk8 to use Adopt-OpenJ9

### LINKED ISSUE #1526: Docker image for jdk8 s390x is using OpenJDK 7 as the boot JVM
Two issues:
1. OpenJDK 7 for s390x doesn't have a JIT so its very slow
2. It doesn't necessarily work to build DDR which has recently been enabled for OpenJ9 zLinux builds.

The docker image can use https://hub.docker.com/r/adoptopenjdk/openjdk8-openj9/ instead

@jdekonin

### PR DESCRIPTION
**Pull request by jdekonin**
* [skip ci]
* fixes #1526

Signed-off-by: Joe deKoning <joe_dekoning@ca.ibm.com>

### CODE REVIEW

File: buildenv/docker/jdk8/s390x/ubuntu16/Dockerfile [hunk 1]
```diff
@@ -22,7 +22,7 @@
 #   docker build -t=openj9 .
 #   docker run -it openj9
 
-FROM ubuntu:16.04
+FROM adoptopenjdk/openjdk8-openj9:latest
```
Comment (pshipton) [inline buildenv/docker/jdk8/s390x/ubuntu16/Dockerfile]: Adding the ppa:openjdk-r/ppa apt-get repo is no longer required.

Comment (pshipton) [reply buildenv/docker/jdk8/s390x/ubuntu16/Dockerfile]: and the comment about it

**Comment by babsingh** [comment]
`adoptopenjdk/openjdk8-openj9:latest` refers to https://github.com/AdoptOpenJDK/openjdk-docker/blob/master/8/jdk/ubuntu/Dockerfile.openj9.

OpenJDK8-OpenJ9 is downloaded from https://api.adoptopenjdk.net/openjdk8-openj9/releases/s390x_linux/latest/binary.

Unzipping OpenJDK8-OpenJ9 creates the following directory `jdk8u162-b12_openj9-0.8.0`.

In `adoptopenjdk/openjdk8-openj9:latest`, environment variable `PATH` is set using `ENV PATH=/opt/java/openjdk/jdk8u152-b16/bin:$PATH`.

`/opt/java/openjdk/jdk8u152-b16/bin` doesn't exist. Instead, `/opt/java/openjdk/jdk8u162-b12_openj9-0.8.0/bin` is created now. So, boot jdk will not be setup properly.

**Comment by babsingh** [comment]
https://github.com/eclipse/openj9/issues/1497 - there is overlapping work in this issue. This issue requests AdoptOpenJDK OpenJ9 builds be used as boot jdk for JDK8 DockerFile(s).

**Comment by babsingh** [comment]
@jdekonin requested to create another fix - https://github.com/eclipse/openj9/pull/1545

**Pull request closed without merging**